ads/_config.js

```diff
@@ -1154,6 +1154,10 @@ const adConfig = jsonConfiguration({
     renderStartImplemented: true,
   },
 
+  'vlyby': {
+    prefetch: 'https://cdn.vlyby.com/amp/qad/qad-outer2.js',
```

Comment:
Have you thought about implmenting `renderStart`? It will give you some performance gains, but up to you.

Reply:
So if I use window.context.renderStart(opt_data) that means that AD SLOT stays collapsed(instead of having fixed height) till I have ad ready ? :thinking: 
https://github.com/ampproject/amphtml/blob/master/ads/README.md

Reply:
you can manually send a signal that says "this ad is ready to be shown" with renderStart. If you don't do this we will delay for a longer time.